src/lib/inspector.js

```diff
@@ -42,19 +45,17 @@ Inspector.prototype = {
 
     this.inspectorCameraEl = document.createElement('a-entity');
     this.inspectorCameraEl.isInspector = true;
-    this.inspectorCameraEl.addEventListener('loaded', function (entity) {
+    this.inspectorCameraEl.addEventListener('loaded', entity => {
```

Comment:
I actually prefer named functions because the name will show up in the stack trace when using the debugger.
```javascript
  this.inspectorCameraEl.addEventListener('loaded', function onLoaded(entity) { ... });
````
In general I don't think arrow functions add any value (many will be probably shocked). It has implicit magic (binding) and makes syntax less tight and redundant because they introduce multiple ways to express one thing (a function expression)